spec/pulsar/client_spec.rb

```diff
@@ -43,9 +43,12 @@
 
     it "can consume a single topic" do
```

Comment:
We shall have a test case for the nil return value as well.

Reply:
that's in the message spec.